### DISCUSSION TITLE
Questions about amchart license

### DISCUSSION THREAD

**Discussion by seyongp** [cloudforet-io/community#120] [General]
Questions about amchart license

I know that cloudforet is an open source.
However, I found that the contents of amchart license in the readme file are stated as paid.
Should I buy an amchart license when I use cloudforet?

<img width="864" alt="image" src="https://user-images.githubusercontent.com/47025380/224940562-70caf3a8-45de-4799-9073-82db21af30b1.png">

**Comment by choonho** [comment]
https://www.amcharts.com/download/

Armcharts is free for personal use. If you want to commercialize Cloudforet, you may have to buy commercial license.
So if you want to test Cloudforet, leave it blank as license value.
